notifications/register/create-binding-server/create-binding-server-apn.5.x.rb

```diff
@@ -0,0 +1,23 @@
+# Download the Ruby helper library from twilio.com/docs/libraries/ruby
+require 'twilio-ruby'
+
+# Get your Account Sid and Auth Token from https://www.twilio.com/console
+account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
+auth_token = 'your_auth_token'
+
+# set up a client to talk to the Twilio REST API
+client = Twilio::REST::Client.new(account_sid, auth_token)
+
+# Retrieve a valid notification service
```

Comment:
Out of curiosity — How you can retrieve an invalid notification service?

I would like to remove this comment as well.